electron/main.js

```diff
@@ -67,42 +67,43 @@ function setStopProcessHandler(app, window, enabled) {
     // So we'll alert the player to close their browser.
     if (global.app_playerOpenedExternalLink) {
       await dialog.showMessageBox({
-        title: 'Bitburner',
-        message: 'You may have to close your browser to properly exit the game.',
-        detail: 'Steam will keep tracking Bitburner as "Running" if any process started within the game is still running.' +
-          ' This includes launching an external link, which opens up your browser.',
-        type: 'warning', buttons: ['OK']
+        title: "Bitburner",
+        message: "You may have to close your browser to properly exit the game.",
+        detail:
+          'Steam will keep tracking Bitburner as "Running" if any process started within the game is still running.' +
+          " This includes launching an external link, which opens up your browser.",
```

Comment:
This look weird, no? It replaced the second part `'` with `"`, but not the first part.

Reply:
Nvm, it's because the first part actually contains `"` as a character, that's mean :)